pkg/sessionctx/variable/tidb_vars.go

```diff
@@ -1002,8 +1001,8 @@ const (
 	TiDBMaxAutoAnalyzeTime = "tidb_max_auto_analyze_time"
 	// TiDBEnableDistTask indicates whether to enable the distributed execute background tasks(For example DDL, Import etc).
 	TiDBEnableDistTask = "tidb_enable_dist_task"
-	// TiDBDDLVersion is used to store tidb ddl version.
-	TiDBDDLVersion = "tidb_ddl_version"
+	// TiDBEnableFastCreateTable indicates whether to enable the fast create table feature.
+	TiDBEnableFastCreateTable = "tidb_enable_fast_create_table"
```

Comment:
Is there a compatibility problem with this? In addition, whether the original variable will become dirty data(not deleted).

Reply:
and since we delete the data when between enable/disable, so I think dirty data will be clear when user switch the variable

Reply:
since 7.6 is a DMR version, therefore, there should be very few clusters that are upgraded from version 7.6.